pkg/scheduler/partition_test.go

```diff
@@ -2804,14 +2804,22 @@ func TestAddTGApplication(t *testing.T) {
 	partition, err := newLimitedPartition(limit)
 	assert.NilError(t, err, "partition create failed")
 	// add a app with TG that does not fit in the queue
-	var tgRes *resources.Resource
-	tgRes, err = resources.NewResourceFromConf(map[string]string{"vcore": "10"})
+	tgRes, err := resources.NewResourceFromConf(map[string]string{"vcore": "10"})
```

Comment:
NIT: this change introduces shadowing for the err variable. Not needed

Reply:
Done